cypress/support/commands.js

```diff
@@ -23,3 +23,16 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+import { userData } from '../support/generate';
+
+Cypress.Commands.add('login', () => {
```

Comment:
Consider adding a parameter to the 'login' command for accepting custom username and password, and use `userData` as default values. This will make the command more flexible for different test cases.